README

```diff
@@ -67,6 +67,11 @@ sudo dnf install ncurses-devel automake autoconf gcc
 sudo pacman -S ncurses automake autoconf gcc
 ~~~
 
+**MacOS/OS X/MacOS X**
```

Comment:
How about just title this as "macOS" and omit the old names of the same OS family?

Reply:
Sure!